lib/rules/no-await-sync-events.ts

```diff
@@ -90,6 +103,11 @@ export default createTestingLibraryRule<Options, MessageIds>({
 			},
 			'AwaitExpression > CallExpression'(node: TSESTree.CallExpression) {
 				const simulateEventFunctionIdentifier = getDeepestIdentifierNode(node);
+				const enabledEventModules = getEnabledEventModules(eventModules);
+				const isFireEventEnabled =
+					enabledEventModules.includes(FIRE_EVENT_OPTION);
+				const isUserEventEnabled =
+					enabledEventModules.includes(USER_EVENT_OPTION);
```

Comment:
These can be inlined

```suggestion
```

Reply:
We're not re-using this variable anywhere at all, so this way we keep the code changes as small as possible